### ISSUE TITLE
Table of contents at side

### ISSUE THREAD

**Issue by TimTaylor** [ukgovdatascience/govdown#24]
It would be great to have the ability to create a sidebar/table of contents with govdown.  For example

![Screenshot 2019-04-30 at 15 40 14](https://user-images.githubusercontent.com/43499035/56970535-3cf0f400-6b5f-11e9-8edf-05f7612aa1b0.png)

**Comment by nacnudus** [comment]
Agreed.  It isn't part of the Design System itself, so we'll have to find the CSS and copy it over.

**Comment by nacnudus** [comment]
This lua filter example might help https://pandoc.org/lua-filters.html#creating-a-handout-from-a-paper

**Comment by nacnudus** [comment]
Noting another request for this feature through another channel.

**Comment by mgriffiths2** [comment]
If this package is still being developed, I would love to see this feature too!

**Comment by nacnudus** [comment]
@mgriffiths2 thanks for your interest.  I currently don't have time at work to develop govdown, so I encourage you to consider [govuk-hugo-r](https://github.com/co-analysis/govuk-hugo-r), which already supports a sidebar of contents.
